qemu/tests/usb_device_check_negative.py

```diff
@@ -0,0 +1,33 @@
+import logging
+
+from virttest import (env_process,
+                      error_context,
+                      virt_vm)
+from qemu.tests.usb_common import parse_usb_topology
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    The usb devices negative test
+
+    1) Boot guest with invalid usb devices
+    2) Verify QEMU error info
+
+    :param test:   QEMU test object.
+    :param params: Dictionary with the test parameters.
+    :param env:    Dictionary with test environment.
+    """
+    # parse the usb topology from cfg
+    parse_usb_topology(params)
+    logging.info("starting vm according to the usb topology")
+    error_info = params["error_info"]
+    error_context.context(("verify [%s] is reported by QEMU..." %
+                          error_info), logging.info)
+    try:
+        env_process.process(test, params, env,
+                            env_process.preprocess_image,
+                            env_process.preprocess_vm)
+    except virt_vm.VMCreateError, e:
```

Comment:
I rem for the speed mismatch error, sometimes it's just a warning, instead of VMCreateError, could you help confirm it in the qemu code?

Reply:
tested on QEMU 1.5.3， they are same.